homeassistant/components/upnp/const.py

```diff
@@ -31,3 +34,118 @@
 ST_IGD_V1 = "urn:schemas-upnp-org:device:InternetGatewayDevice:1"
 ST_IGD_V2 = "urn:schemas-upnp-org:device:InternetGatewayDevice:2"
 SSDP_SEARCH_TIMEOUT = 4
+
+RAW_SENSOR = "raw_sensor"
+DERIVED_SENSOR = "derived_sensor"
+
+SENSOR_ENTITY_DESCRIPTIONS: Dict[str, List[Tuple[SensorEntityDescription, str]]] = {
+    RAW_SENSOR: [
+        (
+            SensorEntityDescription(
+                key=BYTES_RECEIVED,
+                name=f"{DATA_BYTES} received",
+                icon="mdi:server-network",
+                native_unit_of_measurement=DATA_BYTES,
+            ),
+            "d",
```

Comment:
What kind of value is this?

You should subclass `SensorEntityDescription` to `UpnpSensorEntityDescription` and add it to that object

Reply:
It is for the format string for the value returned. 
I was thinking of doing a subclass but wasn't 100% sure. Will do.

Reply:
OK, that is done. Let me know if it looks good. :-)